DFE-Digital/cpd-information
# [1647] Add statuscake monitoring

## Description
Deploy minimal statuscake uptime and SSL monitors

## How to review
- Run domains-plan
- Check statuscake as the 2 monitors have already been created
